fuzz/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "scrypt-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+scrypt = { path = "../scrypt" }
+password-hash = { version = "0.6.0-rc.0", default-features = false, features = ["rand_core"], optional = true }
```

Comment:
You don't need to import `password-hash` explicitly. You can just use `scrypt = { path = "../scrypt", features = ["simple"] }` and then use [its re-export](https://github.com/RustCrypto/password-hashes/blob/master/scrypt/src/lib.rs#L65-L66) from `scrypt`.

Reply:
Sorry, I overlook the simple features in scrypt that re-export the password-hash crate. Fixed now.